code/game/objects/structures/railings.dm

```diff
@@ -22,10 +23,15 @@
 	. = ..()
 	ini_dir = dir
 	if(climbable)
+		flags_1 |= ON_BORDER_1
```

Comment:
Should this be done on initialize?
It seems more like something to be in the object definition.